src/CLI/clitree/cli-xml/ipv4.xml

```diff
@@ -39,6 +39,83 @@ limitations under the License.
         >
     <ACTION builtin="clish_nop"/>
     </COMMAND>
+    
+<!--====================show ip arp <ip-addr> commands ======================-->
+    <COMMAND
+        help="show ARP commands"
+        name="show ip arp">
+        <PARAM
+            name="arp-subcommands"
+            help="ARP subcommands"
+            mode="switch"
+            ptype="SUBCOMMAND"
+            optional="true">
```

Comment:
I don't think we need this optional="true" for mode="switch". And do we actually need this?, since we have only one choice inside switch statement.

Reply:
yes, will remove it